### ISSUE TITLE
Can't install PAT

### ISSUE THREAD

**Issue by eurid527** [CSIRO-Precision-Agriculture/PAT_QGIS_Plugin#58]
I tied to install pat plugin in QGIS 3.16 running on a windows 10 64bit machine. I get and error message, The first lines of message are:

"Couldn't load plugin 'pat' due to an error when calling its classFactory() method 

pkg_resources.DistributionNotFound: The 'pyprecag' distribution was not found and is required by the application 
Traceback (most recent call last):
  File "C:/Users/daneu/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\pat\util\check_dependencies.py", line 421, in check_python_dependencies
    user_path, True)
  File "C:/Users/daneu/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\pat\util\check_dependencies.py", line 519, in create_link
    file_link.Save(link_path, 0)
pywintypes.com_error: (-2147024893, 'The system cannot find the path specified.', None, None)

I know very little about python so any help would be appreciated

Thanks

**Comment by eurid527** [comment]
I found the problem. Microsoft has a cloud storage system "OneDrive". Onedrive allows you to store windows folders such as Desktop, Documents, etc so they are available on any machine you use.

 Any item that is stored to the Desktop cloud folder has a directory address of "C:\USERS\"your machine name"\ONEDRIVE\DESKTOP". It appears PAT did not redirect INSTALL file to this folder or even generate this file. 

I removed DESKTOP from cloud storage and put it back to its default location C:\USERS\"your machine name"\DESKTOP" and reinstalled PAT plugin. INSTALL_PAT....."  was generated on Desktop and plugin was able to be installed following manuals instructions.
